actions/kv_get.yaml

```diff
@@ -12,3 +12,7 @@ parameters:
     type: boolean
     description: 'True if the value is compressed using bzip2 and should be decompressed before returning it'
     required: False
+  decrypt:
+    type: boolean
+    description: 'True if the value is stored as a secret and it should be decrypted before returning it'
+    required: False
```

Comment:
Please add a newline to the end of this file.

https://unix.stackexchange.com/a/18789

Reply:
Other than that, LGTM! 👍